docs/reference/roles_matrix.rst

```diff
@@ -0,0 +1,47 @@
+Roles Matrix
+============
+
+The ``Sonata\UserBundle\Form\Type\RolesMatrixType`` was built to show all roles in a matrix view.
+
+![Roles matrix](../images/roles_matrix.png)
+
+Every admin has defined default roles like:
+
+    - EDIT
+    - LIST
+    - CREATE
+    - VIEW
+    - DELETE
+    - EXPORT
+    - ALL
+
+The roles matrix consists of two parts:
+
+1. shows the matrix with each admin and their permissions.
+2. shows the custom roles which are configured in ``security.yml`` and list them as checkboxes (+ showing their inherited roles).
```

Comment:
Those sentences have no subject, please fix this, also, please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)